gdal/gcore/gdal_rat.cpp

```diff
@@ -592,6 +634,14 @@ CPLXMLNode *GDALRasterAttributeTable::Serialize() const
             CXT_Text, szValue );
     }
 
+/* -------------------------------------------------------------------- */
+/*      Store table type                                                */
+/* -------------------------------------------------------------------- */
+    CPLsnprintf( szValue, sizeof(szValue), "%d", GetTableType() );
```

Comment:
a text representation of the enumeration value would be more readable

Reply:
Is there a good example to look at for this? I presume we'll need a enum<>str function too?

I think we initially used the int approach because `GDALRATFieldUsage` & `GDALRATFieldType` did the same.

Reply:
I was thinking to just doing it at hand, since it would be used just there.